lib/dry/web/roda/skeletons/umbrella/Gemfile

```diff
@@ -0,0 +1,40 @@
+source "https://rubygems.org"
+
+gem "rake"
+
+# Web framework
+gem "dry-component"
+gem "dry-web"
+gem "dry-web-roda"
```

Comment:
Thanks for the bin/console reminder @gotar  , I'll fix that now.

I plan to make new releases of all the gems tonight, so the generated Gemfile will not need to use master (apart from rom-rb gems).